<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtVTKRenderWindows</class>
 <widget class="QMainWindow" name="QtVTKRenderWindows">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtVTKRenderWindows</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/Icons/help.png</normaloff>:/Icons/help.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>22</width>
    <height>22</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>661</width>
      <height>571</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="QVTKOpenGLWidget" name="view2" native="true"/>
     </item>
     <item row="0" column="1">
      <widget class="QVTKOpenGLWidget" name="view4" native="true"/>
     </item>
     <item row="1" column="1">
      <widget class="QVTKOpenGLWidget" name="view3" native="true"/>
     </item>
     <item row="0" column="0">
      <widget class="QVTKOpenGLWidget" name="view1" native="true"/>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>100</y>
      <width>201</width>
      <height>521</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QCheckBox" name="resliceModeCheckBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Oblique Reslice</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="thickModeCheckBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Thick Slab</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="blendModeGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>181</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Blend mode</string>
     </property>
     <widget class="QRadioButton" name="radioButton_Min">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Min Intensity Blend</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_Max">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Max Intensity Blend</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_Mean">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Mean Blend</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="resetButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>250</y>
       <width>191</width>
       <height>211</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="AddDistance1Button">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Distance On View 1</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="DcmDIr">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>25</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>DcmDir</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="m_dcmDIR">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>641</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <action name="actionOpenFile">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/Icons/fileopen.png</normaloff>:/Icons/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>Open File...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset>
     <normaloff>:/Icons/print.png</normaloff>:/Icons/print.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset>
     <normaloff>:/Icons/help.png</normaloff>:/Icons/help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>:/Icons/filesave.png</normaloff>:/Icons/filesave.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLWidget</class>
   <extends>QWidget</extends>
   <header>QVTKOpenGLWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
